src/reduceHandlers.js

```diff
@@ -1,33 +1,66 @@
 export const someFatalHandlers = (...handlers) => {
   return (error, action) => {
-    let data
-    handlers.some((handler) => {
-      data = handler(error, action)
-      return typeof data !== 'undefined'
```

Comment:
If `handler` is `async () => { return undefined }`, `data` will be a `Promise` that resolves to `undefined`.